app/lib/services/report.rb

```diff
@@ -56,7 +56,6 @@ def headers
         funding_choice
         funding_eligiblity_status_code
         targeted_delivery_funding_eligibility
-        works_in_nursery
```

Comment:
When removing data i guess we have to inform a team about this? @Lockyy

Reply:
Yes, we should inform the lead provider team that going forward this data won't be populated for new applications.

Reply:
Also who is in the lead providers team?